group/assignment5/Graph.java

```diff
@@ -0,0 +1,218 @@
+package assignment5;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
+import java.util.Collections;
+
+
+/**
+ * This class represents a directed graph data structure.
+ * A graph can be described as a set of vertecies. In case graph
+ * is directed each vertex has its indegree and outdegree vertecies.
+ * This class helps you to construct a directed graph and operate on it

```

Comment:
I think this part is redundant:
"This class helps you to construct a directed graph and operate on it easily."

Reply:
Done